### ISSUE TITLE
Allow choosing position of custom column in tables

### ISSUE THREAD

**Issue by aptkingston** [Budibase/budibase#5005]
Currently when adding child components to tables, they will always be inserted as the first column. It could be useful to allow choosing via a table setting whether this column should be the first or last column.

Mentioned in https://github.com/Budibase/budibase/discussions/4999

**Comment by ggs331** [comment]
The same request, but it's better the child components can be placed any column in the table instead of just first or last column. I have similar topics on #4889

**Comment by kasgithub8** [comment]
@aptkingston Any updates on implementing this?

**Comment by Simoon-F** [comment]
Is there an update?

**Comment by kasgithub8** [comment]
@Simoon-F

yes, there should be a new plugin released this month. It's gonna be something like "Super Table" where you can customize the table.
